xendit-java-lib/src/main/java/com/xendit/model/Disbursement.java

```diff
@@ -348,4 +366,55 @@ private static DisbursementClient getClient() {
   private static boolean isApiKeyExist() {
     return Xendit.apiKey != null;
   }
+
+  /**
+   * Create v1 disbursement with required parameters
+   *
+   * @param xendit_idempotency_key
+   * @param reference_id
+   * @param currency
+   * @param channel_code
+   * @param account_name
+   * @param account_number
+   * @param description
+   * @param amount
+   * @return Disbursement
+   * @throws XenditException
+   */
+  public static Disbursement create(
```

Comment:
can add tests for changes on this file ya?

Reply:
daa894930253d1c03ff37f2ff12aa109ea534837